qiskit_nature/algorithms/ground_state_solvers/adapt_vqe.py

```diff
@@ -248,6 +284,11 @@ def solve(
                         gradlog += "\t(*)"
                 logger.info(gradlog)
             if np.abs(max_grad[0]) < self._threshold:
+                if iteration == 1:
+                    raise QiskitNatureError(
+                        "Gradient choice is not suited as it leads to all non-zero gradients. "
+                        "Try a different gradient method."
+                    )
```

Comment:
I think this error message here should says it `leads to alls-zero gradients` right rather than non-zero. That would then match too with the text in Raises docstring above.